experimenter/bin/should-pr.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+import subprocess
+import sys
+from pathlib import Path
+
+MANIFEST_DIR = (
+    Path(__file__)
+    .parent.parent.joinpath("experimenter", "features", "manifests")
+    .resolve()
+)
+
+
+def changed_files() -> list[Path]:
+    output = subprocess.check_output(["git", "status", "--porcelain"], encoding="utf-8")
+    files = [Path(line[3:]) for line in output.splitlines()]
+    return files
+
+
+def is_ref_cache(path: Path):
+    return path.name == ".ref-cache.yaml" and path.parent.parent.resolve() == MANIFEST_DIR
```

Comment:
If we end up moving paths around that `.parent.parent` might break, would it be better just do do like `MANIFEST_DIR in path` or something?